cmd/jet/main.go

```diff
@@ -126,3 +164,12 @@ func printErrorAndExit(error string) {
 	flag.Usage()
 	os.Exit(-2)
 }
+
+func detectSchema(dsn string) (source string) {
+	schemeRe := regexp.MustCompile(`^(.+)://.*`)
+	match := schemeRe.FindStringSubmatch(dsn)
+	if len(match) < 2 { // not found
+		return ""
+	}
+	return match[1]
```

Comment:
Can it be done without regex? Split over `//` and return first part if exist.

Reply:
done. + add test.